docs/index.md

```diff
@@ -3,7 +3,7 @@ lang: en-us
 title: Home
 home: true
 actions:
-  - text: "Get Started 💡"
+  - text: "Download 💡"
```

Comment:
Do we want the download link to launch to the manual? Surely this should go to a separate download page?

Reply:
I'm waiting on a pr to generate the link for this. @mauricioszabo can you merge that alias pr for me? Trying not to step on toes, but that helps us over here.